pysat/instruments/methods/sw.py

```diff
@@ -511,3 +511,74 @@ def calc_daily_Ap(ap_inst, ap_name='3hr_ap', daily_name='Ap',
     ap_inst.meta.__setitem__(daily_name, meta_dict)
```

Comment:
This could also be modified, though technically not in pull.
ap_inst.meta[daily_name] =  meta_dict